VMEncryption/main/handle.py

```diff
@@ -971,6 +971,22 @@ def handle_encryption(public_settings, encryption_status, disk_util, bek_util, e
                 logger.log('Calling enable for volume type {0}.'.format(volume_type))
                 enable_encryption()
 
+def _is_confidential_temp_disk_encryption(self,logger):
```

Comment:
Put a comment explaining what this function does.

Reply:
done